source/fundamentals/crud/restful-api-tutorial.txt

```diff
@@ -0,0 +1,384 @@
+.. _csharp-crud-restful-api-tutorial:
+
+=====================================================================
+Tutorial: Create a RESTful API with the {+driver-short+} Driver
+=====================================================================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: crud, code example, tutorial
+   :description: Learn how to use the .NET/C# Driver to create a RESTful API for your application.
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+In this tutorial, you will learn how to create a RESTful API with endpoints that
+perform basic create, read, update, and delete (CRUD) operations across MongoDB
+Atlas.
+
+Prequisites
+-----------
+
+To complete this tutorial, you must have the following:
+
+- An `Atlas account
+  <https://account.mongodb.com/account/register?tck=docs_atlas>`__ with a
+  deployed and configured MongoDB Atlas cluster that is M0 or higher.
```

Comment:
S: instead of linking to atlas signup, link here: https://www.mongodb.com/docs/atlas/getting-started/

eg
Set up a MongoDB Atlas account and deploy a cluster. To view instructions, see ...